support/linux/provision.sh

```diff
@@ -14,24 +14,28 @@ pushd /src
 cp components/hab/install.sh /tmp/
 sh support/linux/install_dev_0_ubuntu_latest.sh
 sh support/linux/install_dev_9_linux.sh
-echo 'eval "$(direnv hook bash)"' >> /root/.bashrc
-echo "source /src/support/linux/helpers.sh" >> /root/.bashrc
+# shellcheck disable=SC2016
+echo 'eval "$(direnv hook bash)"' | sudo tee --append /root/.bashrc > /dev/null
+echo "source /src/support/linux/helpers.sh" | sudo tee --append /root/.bashrc > /dev/null
 . ~/.profile
 . ~/.bashrc
 . ./.envrc
 
-hab pkg install core/postgresql core/shadow -c stable
-hab pkg install core/builder-datastore -c unstable
-hab pkg install core/docker
-hab pkg install core/hab-pkg-export-docker
+sudo hab pkg install core/postgresql core/shadow -c stable
+sudo hab pkg install core/hab-sup core/hab-launcher
+sudo hab pkg install core/builder-datastore
+sudo hab pkg install core/docker
+sudo hab pkg install core/hab-pkg-export-docker
```

Comment:
Added a couple additional packages here that logs were complaining about missing